src/tunacode/utils/system/gitignore.py

```
@@ -1,35 +1,37 @@
-    except FileNotFoundError:
-        return None
-    except Exception as e:
-        print(f"Error reading {filepath}: {e}")
-        return None
-    patterns.add(GIT_DIR_PATTERN)
-    return patterns
+def _is_fast_excluded(relative_path: Path) -> bool:
+    return any(part in DEFAULT_EXCLUDE_DIRS for part in relative_path.parts)
+
+
+def _matches_ignored_path(relative_path: Path, spec: pathspec.PathSpec, is_dir: bool) -> bool:
```

Ruff (0.15.7)

[warning] Boolean-typed positional argument in function definition

(FBT001)